packages/core/src/ceramic.ts

```diff
@@ -231,6 +231,7 @@ export class Ceramic implements StreamReaderWriter, StreamStateLoader {
   private _signer: CeramicSigner
   public readonly dispatcher: Dispatcher
   public readonly loggerProvider: LoggerProvider
+  public readonly recon: IReconApi
```

Comment:
um if we role back the node metrics changes we probably don't want to roll this stuff back?

Reply:
why wouldn't we?  We only want this for NodeMetrics anyway.